Automattic/jetpack
# AI Extension: add simple button color animation when requesting

This PR adds a simple winking effect to the toolbar button of the AI Assistant extension to show a hint about the requesting/suggesting state.
We used this method to show the state because we wouldn't like to change the block representation as it in the editor canvas.

Fixes #

## Proposed changes:

* AI Extension: add simple toolbar button color animation when requesting

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor
* Add some content to the post by using a paragraph block
* Edit the content by using the AI extension
* Confirm you see the winking effect when the app is requesting/suggesting content
* Confirm the rest of the implementation works as expected


https://github.com/Automattic/jetpack/assets/77539/8694f773-0aab-4cc6-834e-92f52893983e
